src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.targets

```diff
@@ -83,7 +114,32 @@ Copyright (C) Sayed Ibrahim Hashimi, Chuck England 2011-2013. All rights reserve
     <SlowCheetah.TransformTask Source="@(ScFilesToTransform->'%(SourceFile)')"
                               Transform="%(TransformFile)"
                               Destination="%(DestinationFile)"
-                              Condition=" Exists('%(TransformFile)')"/>
+                              Condition="Exists('%(TransformFile)')"/>
+
+    <ItemGroup>
+      <!--Gather each category of files to transform again since more may have been added between targets-->
+      <_ScNoneFilesToTransform Include="@(ScFilesToTransform)" Condition="('%(BuildAction)' == '' or '%(BuildAction)' == 'None') and Exists('%(DestinationFile)')" />
+      <_NoneWithTargetPath Remove="@(_ScNoneFilesToTransform)" />
+      <_NoneWithTargetPath Include="@(_ScNoneFilesToTransform->'%(DestinationFile)')" />
+
+      <_ScContentFilesToTransform Include="@(ScFilesToTransform)" Condition="'%(BuildAction)' == 'Content' and Exists('%(DestinationFile)')" />
+      <ContentWithTargetPath Remove="@(_ScContentFilesToTransform)" />
+      <ContentWithTargetPath Include="@(_ScContentFilesToTransform->'%(DestinationFile)')" />
+
+      <_ScResourceFilesToTransform Include="@(ScFilesToTransform)" Condition="'%(BuildAction)' == 'Resource' and Exists('%(DestinationFile)')" />
+      <Resource Remove="@(_ScResourceFilesToTransform)" />
+      <Resource Include="@(_ScResourceFilesToTransform->'%(DestinationFile)')" />
```

Comment:
I'm unsure if this item group is even used at this point. From what I've seen, it's only `ContentFilesProjectOutputGroup` from the Cpp targets that uses it, in unknown circumstances. In that case, we can add to `ContentFilesProjectOutputGroupDependsOn`, either in the main targets or a new target file for cpp projects.

Reply:
Transform already doesn't depend on `ContentFilesProjectOutputGroupDependsOn`, so it could already be broken with resource items. Leave it as is for now and open a different issue to fix resource items, and to evaluate if support for them is even needed.